website/content/docs/agent-and-proxy/agent/apiproxy.mdx

```diff
@@ -1,62 +1,62 @@
 ---
 layout: docs
-page_title: Vault Agent API Proxy
+page_title: OpenBao Agent API Proxy
 description: >-
-  Vault Agent's API Proxy functionality allows you to use Vault Agent's API as a proxy
-  for Vault's API.
+  OpenBao Agent's API Proxy functionality allows you to use OpenBao Agent's API as a proxy
+  for OpenBao's API.
 ---
 
-# Vault agent API proxy
+# OpenBao agent API proxy
 
-Vault Agent's API Proxy functionality allows you to use Vault Agent's API as a proxy
-for Vault's API.
+OpenBao Agent's API Proxy functionality allows you to use OpenBao Agent's API as a proxy
+for OpenBao's API.
 
 ~> Note: This functionality will be deprecated in a future release. Please
-switch to using [Vault Proxy](/vault/docs/agent-and-proxy/proxy) for API proxying purposes, instead.
+switch to using [OpenBao Proxy](/vault/docs/agent-and-proxy/proxy) for API proxying purposes, instead.
```

Comment:
I don't know if/when we'll also want to change the path here and in subsequent locations.

Reply:
Until now I left the path names intact on purpose. This allows us to still build the documentation until we integrated it into the new website.

Reply:
We may want to consider adding a redirect: 

https://github.com/openbao/openbao/blob/8dead566a8ab017c97a6417df33d326167eb5f57/website/redirects.js#L88-L92

That way anyone wanting to replace the URLs and compare between upstream and us can do so easily.